Sources/Afluent/SequenceOperators/OutputSequence.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  OutputSequence.swift
+//  Afluent
+//
+//  Created by Roman Temchenko on 2025-03-05.
+//
+
+import Foundation
+
+extension AsyncSequences {
+    public struct OutputAt<Upstream: AsyncSequence & Sendable>: AsyncSequence, Sendable {
+        public typealias Element = Upstream.Element
+        let upstream: Upstream
+        let index: Int
+
+        public struct AsyncIterator: AsyncIteratorProtocol {
+            var upstreamIterator: Upstream.AsyncIterator
+            let index: Int
+            var nextIndex = 0
+
+            public mutating func next() async throws -> Element? {
+//                try Task.checkCancellation()
+                guard nextIndex <= index else { return nil }
+                while let next = try await upstreamIterator.next() {
+                    Swift.print("iter \(nextIndex)")
+//                    try Task.checkCancellation()
```

Comment:
I know you're working on it. I'm leaving this as a reminder in GitHub before I click "merge" that these commented lines should be removed.